README.md

```diff
@@ -1,9 +1,13 @@
 # Binance order book metrics challenge
 
 ## Overview
-Challenge program gives interested individuals a chance to prove themselves and learn a bit about
-the company & products. Our challenges are extremely independent and will require you to manage 
-your own time and work process.
+[Challenge program](https://github.com/1712n/challenge) gives interested individuals a chance to prove themselves and learn a bit about
+the company & products. Our challenges are extremely independent and will require you to manage your own time and work process.
+
+### Process
+1. Submit your challenge as described in [Workflow](README.md#workflow).
+1. In case your submission is approved, one of our team members will reach out to set up 30-minute Google Meet call.
```

Comment:
We told about HR/Bar Raiser 15-30m and the following 1-1.5h technical interviews